### ISSUE TITLE
Bootstrap servers seems to stop after an error

### ISSUE THREAD

**Issue by AurelienFT** [massalabs/massa#3932]
One of our bootstrap server have this logs : 
```
2023-05-11T09:29:49.082634Z  INFO massa_bootstrap::server: bootstrapped peer [::ffff:173.212.193.122]:53088
2023-05-11T09:29:53.437678Z  INFO massa_bootstrap::server: bootstrapped peer [::ffff:184.174.34.65]:47658
2023-05-11T09:29:54.890168Z  INFO massa_bootstrap::server: bootstrapped peer [::ffff:173.212.194.120]:45618
2023-05-11T09:29:55.038266Z  INFO massa_bootstrap::server: bootstrapped peer [::ffff:75.119.137.36]:34142
2023-05-11T09:29:55.720178Z ERROR massa_bootstrap::server: bootstrap listener error: all io errors except for Timedout, and would-block (unix error when timed out)
2023-05-11T09:29:56.392776Z  INFO massa_bootstrap::server: bootstrapped peer [::ffff:194.60.201.37]:34336
2023-05-11T09:30:08.320367Z  INFO massa_bootstrap::server: bootstrapped peer [::ffff:194.163.151.185]:40196
```
After this it didn't bootstrapped any new peers since 20 minutes. So I think the error has stopped the bootstrap server but it shouldn't. It should just stop the bootstrap with this.
@Ben-PH Can you investigate this in prio as it seems to make our bootstrap servers go to stuck.

**Comment by AurelienFT** [comment]
Ok I think I known what is it related to. It's because there is no file descriptors open because of the network module. Will upgrade the fd count on our servers. Just need a more informative message so maybe a duplicate a #3927 . In this case you can close it.

**Comment by Ben-PH** [comment]
fix: #4113
